### DISCUSSION TITLE
Interactive Graph / Backlink

### DISCUSSION THREAD

**Discussion by adrien1212** [McShelby/hugo-theme-relearn#975] [Ideas]
Hi, 

To improve the navigation and learning between related concepts in large site, it would be amazing to have an “interactive graph” like https://amethyst.bencuan.me/

![image](https://github.com/user-attachments/assets/d4b1a743-6eb1-4212-9278-cffc4da9e806)

PS : Thank you for your work <3

**Comment by McShelby** [comment]
Personally, I don't see a gain of such a diagram for documentation related sites.
